.github/workflows/release.yaml

```diff
@@ -0,0 +1,48 @@
+name: release
+on:
+  push:
+    tags:
+      - 'v*'
+env:
+  go-version: 1.16
+jobs:
+  build:
+    name: Release
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      
+      - name: Setup golang
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ env.go-version }}
+
+      - name: Build binaries
+        run: make install GOBIN=`pwd`/docker
+
+      - name: Build setup-hw image
+        run: |
+          docker build -t quay.io/cybozu/setup-hw:latest --target stage1 docker
+          docker build -t quay.io/cybozu/setup-hw-secret:latest docker
+
+      - name: Push images
+        run: |
+          echo ${{ secrets.QUAY_PASSWORD }} | docker login -u ${{ secrets.QUAY_USER }} --password-stdin quay.io
+          tag=${GITHUB_REF#refs/tags/v}
+          for name in setup-hw setup-hw-secret; do
+              echo "pushing ${name}:v${tag} ..."
+              docker tag quay.io/cybozu/${name}:latest quay.io/cybozu/${name}:$tag
+              if echo $tag | grep -q -e '-'; then docker push quay.io/cybozu/${name}:latest; fi
```

Comment:
You are still pushing the pre-release image.
https://cybozu.slack.com/archives/C93NC5UR2/p1621316251012100